src/uipath/_cli/_evals/_runtime.py

```diff
@@ -417,6 +427,18 @@ async def _execute_eval(
 
         return evaluation_run_results
 
+    async def _generate_input_for_eval(
+        self, eval_item: EvaluationItem
+    ) -> EvaluationItem:
+        """Use LLM to generate a mock input for an evaluation item."""
+        if not self.context.entrypoint:
+            raise UiPathInputMockingError("entrypoint must be provided for eval runs")
+
+        input_schema = extract_input_schema_from_entrypoint(self.context.entrypoint)
```

Comment:
Not sure this is a good idea. It's better to extract from the agent definition. Can we somehow ensure the evaluation starts with an `AgentDefinition`? Maybe we want a `CodedAgentDefinition` (similar to `LowCodeAgentDefinition`)